gen/enum.go

```diff
@@ -22,17 +22,49 @@ package gen
 
 import "github.com/thriftrw/thriftrw-go/compile"
 
+// enumGenerator generates code to serialize and deserialize enums.
+type enumGenerator struct {
+	hasReaders
+}
+
+func (e *enumGenerator) Reader(g Generator, spec *compile.EnumSpec) (string, error) {
```

Comment:
Good idea. Created https://github.com/thriftrw/thriftrw-go/pull/85 with just these generator changes. Will rebase this PR on top.